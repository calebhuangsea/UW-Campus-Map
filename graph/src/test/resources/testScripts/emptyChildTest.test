# Test empty child file
# Tests that a graph can be created
# a node can be added to the graph
# ListChildren behaves correctly

# Create a graph
CreateGraph graph1

# Add two nodes
AddNode graph1 n1

# Print the nodes in the graph and the outgoing edges from n1
ListNodes graph1
ListChildren graph1 n1