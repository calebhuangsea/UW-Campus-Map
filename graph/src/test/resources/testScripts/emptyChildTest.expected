# Test empty child file
# Tests that a graph can be created
# a node can be added to the graph
# ListChildren behaves correctly

# Create a graph
created graph graph1

# Add two nodes
added node n1 to graph1

# Print the nodes in the graph and the outgoing edges from n1
graph1 contains: n1
the children of n1 in graph1 are: